pom.xml

```diff
@@ -1001,6 +1001,49 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.openrewrite.maven</groupId>
+                <artifactId>rewrite-maven-plugin</artifactId>
+                <version>5.14.1</version>
+                <configuration>
+                    <checkstyleConfigFile>.checkstyle/checkstyle.xml</checkstyleConfigFile>
+                    <activeRecipes>
+                        <recipe>io.strimzi.strimzi-kafka-operator</recipe>
+                    </activeRecipes>
+                    <exclusions>
+                        <exclusion>.git/**</exclusion>
+                        <exclusion>**/*.yaml</exclusion>
+                        <exclusion>**/*.yml</exclusion>
+                        <exclusion>**/*.md</exclusion>
+                        <exclusion>**/*.json</exclusion>
+                        <exclusion>**/*.xml</exclusion>
+                        <exclusion>**/*.properties</exclusion>
+                        <exclusion>.jenkins/**</exclusion>
+                        <exclusion>.github/**</exclusion>
+                        <exclusion>.checksums</exclusion>
+                        <exclusion>*.version</exclusion>
+                        <exclusion>*.os</exclusion>
+                        <exclusion>*.adoc</exclusion>
+                        <exclusion>*.txt</exclusion>
+                    </exclusions>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.openrewrite.recipe</groupId>
+                        <artifactId>rewrite-static-analysis</artifactId>
+                        <version>1.1.1</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>validate</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                    </execution>
+                </executions>
```

Comment:
Rather than execute the plugin always (which could take a while) it might be better to move the plugin to a profile that you can activate separately; that saves folks some time, and does not make further code changes in the validate phase.

Reply:
Okay. What do you think @scholzj?

Reply:
Yes, we want it to run only manually. So that sounds fine.

Reply:
Done :+1: